packages/cells/src/widget.ts

```diff
@@ -19,6 +19,10 @@ import {
   PanelLayout, Panel, Widget
 } from '@phosphor/widgets';
 
+import {
```

Comment:
This `import` is in the wrong place alphabetically, and it looks like the other imports that were in this file before this PR were also not alphabetical, could you please update them since you're already in this file?